database/models.py

```diff
@@ -12,27 +16,31 @@ class Author(db.Document):
     updatedOn = db.DateTimeField(default=datetime.datetime.utcnow)
     meta = {"collection": "authors"}
 
-    def to_json(self):
+    def to_json(self, *args, **kwargs):
         obj = self.to_mongo()
-        return objectToJson(obj)
+        return object_to_json(obj)
 
 
 class ScrapedAuthor(db.Document):
+    """Scraped Author Model"""
+
     name = db.StringField()
-    DOB = db.DateTimeField()
+    dob = db.DateTimeField()
     country = db.StringField()
     description = db.StringField()
     link = db.StringField()
     createdOn = db.DateTimeField(default=datetime.datetime.utcnow)
     updatedOn = db.DateTimeField(default=datetime.datetime.utcnow)
     meta = {"collection": "scrapedAuthors"}
 
-    def to_json(self):
+    def to_json(self, *args, **kwargs):
```

Comment:
Do we make use of *args, **kwargs here? I don't think so. If not, then remove these.